include/core/grids.h

```diff
@@ -72,6 +72,7 @@ read_mesh_and_manifolds(
                                                           triangulation,
   const Parameters::Mesh &                                mesh_parameters,
   const Parameters::Manifolds &                           manifolds_parameters,
+  const bool &                                            restart,
```

Comment:
No criticism intended here, but why is this inserted in the middle?

Reply:
My mood of the day  :p.
Honestly, I don't know. I think this is because I feel restart is a parameter as mesh_parameters and manifolds_parameters so I put it straight after them even if not the same object after all.